english/explore-outside/talk-child/talk-more/nice-talk/nice-talk.md

```diff
@@ -1,3 +1,7 @@
 The child says, "Oh OK, so sorry for trapping you. Must have been a misunderstanding."
 
-"Here's $500 and the keys to my car."
```

Comment:
This deletion doesn't seem like it's in the spirit of the repo to me. If you just leave the line in, though, it seems like the rest of your additions could still follow from that.